### PR TITLE
Support automation account managment

### PR DESCRIPTION
**Pull request by yuwzho**
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

azure_rm_automationaccount
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_automationaccount.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_automationaccount
+version_added: "2.9"
+short_description: Manage Azure automation account.
+description:
+    - Create, delete an automation account.
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_automationaccount.py]: Azure

**Review by yungezz — APPROVED** [review]
LGTM, minor issue.

**Comment by yuwzho** [comment]
@mattclay Could you please register the `Microsoft.automation`?

**Comment by mattclay** [comment]
`Microsoft.Automation` has been registered.

**Comment by yuwzho** [comment]
ready_for_review

The failed test is not related to this PR, can be re-trigger before merge or new coming commit

**Comment by Fred-sun** [comment]
@yungezz @zikalino review_need

**Comment by Fred-sun** [comment]
@yungezz @zikalino Please help to review this PR when you're available! Thanks a lot!

**Comment by Fred-sun** [comment]
@yungezz @zikalino @MyronFanQiu review_need

**Comment by Fred-sun** [comment]
ready_for_review

**Review by nitzmahone — APPROVED** [review]

**Comment by Fred-sun** [comment]
@yungezz @yuwzho @MyronFanQiu This PR already approved. Now this PR show exist conflicts, but I don't see the cause of the conflict. Could you take a look this and promote to merge? Thank you very much!

**Pull request merged**